contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -3219,21 +3187,22 @@ class tsqlMutator : public TSqlParserBaseListener
 			}
 		}
 	}
-	
-	// NB: this is copied in tsqlBuilder
+
+	// NB: the following are copied in tsqlBuilder
 	void exitColumn_def_table_constraints(TSqlParser::Column_def_table_constraintsContext *ctx)
 	{
 		handleTableConstraintWithoutComma(ctx);
-	}
-	
-	// NB: this is copied in tsqlBuilder
+  }
 	void exitConstant(TSqlParser::ConstantContext *ctx) override
 	{
 		// Check for floating-point number without exponent
 		handleFloatWithoutExponent(ctx);
 	}
-
-// NB: this is copied in tsqlBuilder
+	void exitPredicate(TSqlParser::PredicateContext *ctx) override
```

Comment:
Why this function implement write twice ?

Reply:
see comment below